base/src/main/java/io/spine/code/proto/ref/InPackage.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.code.proto.ref;
+
+import com.google.protobuf.Descriptors.Descriptor;
+import io.spine.code.proto.PackageName;
+
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Reference to all message types in a proto package.
+ */
+final class InPackage extends AbstractTypeRef {
+
+    private static final long serialVersionUID = 0L;
+
+    /**
+     * A suffix for referencing all types in a proto package.
+     */
+    private static final String WILDCARD = ".*";
+
+    /**
+     * The name of the referenced proto package.
+     */
+    private final PackageName packageName;
+
+    static Optional<TypeRef> parse(String value) {
+        checkNotNull(value);
+        if (value.endsWith(WILDCARD)) {
```

Comment:
Is `.*` should be allowed? Will it mean that we are referencing all message types within all possible Protobuf packages?